### DISCUSSION TITLE
Как пользоваться ByeDPI на ПК Windows? Куда что писать?

### DISCUSSION THREAD

**Discussion by chemodanp** [hufrea/byedpi#238] [General]
Помогите, пожалуйста, понять куда вообще нужно писать все эти циферки и буковки. Я попробовал позапускать разное, пишет "GoodByeDPI is now running". А где менять настройки под себя? В строку ничего не даёт писать. 
Простите за глупый вопрос, вообще не смыслю в этом, а ютуба хочется.

**Comment by edvejak** [comment]
You may start byedpi as proxy

**Comment by vehametra** [comment]
в свойствах ярлыка было поле под опции командной строки.
